tests/test_fitsChan.py

```diff
@@ -74,19 +74,26 @@ def test_FitsChanGetFits(self):
         fv = fc.getFitsS("FRED")
         self.assertTrue(fv.found)
         self.assertEqual(fv.value, "99")
+        self.assertEqual(fc.getNcard(), 1)
+        self.assertEqual(fc.getCardNames(), ["FRED"])
+        # replace this card
         fc.setFitsF("FRED1", 99.9, "Hello there", True)
         fv = fc.getFitsS("FRED1")
         self.assertTrue(fv.found)
         self.assertEqual(fv.value, "99.9")
-        fc.setFitsCF("FRED2", complex(99.9, 99.8), "Hello there", True)
-        fv = fc.getFitsCF("FRED2")
+        self.assertEqual(fc.getNcard(), 1)
+        self.assertEqual(fc.getCardNames(), ["FRED1"])
+        fc.setFitsCF("FRED1", complex(99.9, 99.8), "Hello there", True)
+        fv = fc.getFitsCF("FRED1")
```

Comment:
I felt the test didn't need all those extra cards. The first new card was informative (e.g. FRED1 and FRED2), but the rest (FRED3 and FRED4) did not enhance the test, and reusing a name is also a useful test.